tensorflow/core/ops/experimental_dataset_ops.cc

```diff
@@ -441,6 +441,14 @@ REGISTER_OP("ExperimentalIgnoreErrorsDataset")
     .Attr("output_shapes: list(shape) >= 1")
     .SetShapeFn(shape_inference::ScalarShape);
 
+REGISTER_OP("IgnoreErrorsDatasetV2")
```

Comment:
Can we make this a new attribute on IngoreErrorsDataset with a default value of False? That is technically not what we always recommend (we prefer inputs) but it feels that this is particularly unlikely to vary at runtime.

Reply:
Is this comment addressed to @tensorflow/api-owners?